kafka/conn.py

```diff
@@ -752,6 +753,24 @@ def _next_correlation_id(self):
         self._correlation_id = (self._correlation_id + 1) % 2**31
         return self._correlation_id
 
+    def _check_version_above_0_10(self, response):
+        test_cases = [
+            # format (<broker verion>, <needed struct>)
```

Comment:
might note that the order here matters, and/or make sure we reverse sort it before checking for best match below